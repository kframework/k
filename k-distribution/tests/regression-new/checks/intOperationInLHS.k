module INTOPERATIONINLHS
  imports INT

syntax KItem ::= "foo" "(" Int ")"

rule foo(A +Int 1) => .K

endmodule
