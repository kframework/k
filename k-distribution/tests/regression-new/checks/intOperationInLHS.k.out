[Error] Compiler: Illegal function symbol _+Int_ on LHS of rule. Consider
adding `simplification` attribute to the rule if this is intended.
	Source(intOperationInLHS.k)
	Location(7,10,7,18)
[Error] Compiler: Illegal function symbol foo(_)_INTOPERATIONINLHS_Int_Int on
LHS of rule. Consider adding `simplification` attribute to the rule if this is
intended.
	Source(intOperationInLHS.k)
	Location(17,10,17,16)
[Error] Compiler: Illegal function symbol foo(_)_INTOPERATIONINLHS_Int_Int on
LHS of rule. Consider adding `simplification` attribute to the rule if this is
intended.
	Source(intOperationInLHS.k)
	Location(19,19,19,25)
[Error] Compiler: Had 3 structural errors.
